server/src/main/java/org/opensearch/cluster/service/ClusterApplierService.java

```diff
@@ -121,7 +128,17 @@ public class ClusterApplierService extends AbstractLifecycleComponent implements
 
     private NodeConnectionsService nodeConnectionsService;
 
-    public ClusterApplierService(String nodeName, Settings settings, ClusterSettings clusterSettings, ThreadPool threadPool) {
+    private final Histogram clusterStateAppliersHistogram;
+
+    private final Histogram clusterStateListenersHistogram;
+
+    public ClusterApplierService(
```

Comment:
I have refactored the UTs to use ClusterServiceUtils class to create clusterService object which is being used in majority of the UT changes. This will help in avoiding changes to such UTs in future if a new parameter say TraceRegistry is introduced in ClusterService constructor. Please see if this new change addresses the concern.